src/components/Calculator.js

```diff
@@ -0,0 +1,86 @@
+function App() {
+  return (
+    <div className="container my-4">
+      <div className="calculator card">
+        <input
+          type="text"
+          className="calculator-screen z-depth-1"
+          value=""
+          disabled
+        />
+
+        <div className="calculator-keys">
+          <button type="button" className="operator btn btn-info" value="+">
```

Comment:
- [ ] Kindly make this component a class-based component. For this project, you're to write your components in class-based so later you'll refactor to functional components. Please refactor this functional component to a class-based component 👍